TCPEchoServer/Client.cpp

```diff
@@ -0,0 +1,51 @@
+﻿#include "Socket.h"
+#include <iostream>
+
+int main(int argc, char* argv[]) {
+
+	WSADATA wsaData;
+
+	std::cout << "Input your name: ";
+	std::string s;
+	std::cin >> s;
+
+	int iResult = WSAStartup(MAKEWORD(2, 2), &wsaData);
+	if (iResult != 0)
+	{
+		std::cout << "WSAStartup failed: " << iResult << std::endl;
+	}
+
+	{
+		Socket socket{};
+		if (argc == 0)
+		{
+			std::cout << "No command line arguments, please, define " << std::endl;
```

Comment:
Please create a 'usage' function and call it here

Reply:
Used